src/re_pressed/impl.cljs

```diff
@@ -175,7 +172,7 @@
                        :ctrlKey  (.-ctrlKey e)
                        :metaKey  (.-metaKey e)
                        :shiftKey (.-shiftKey e)
-                       :keyCode  (.-keyCode e)}
+                       :key      e-key}
```

Comment:
storing `:key` in the map instead of `:keyCode`. I belatedly realized that there is possibly a way of using both mechanisms, ie both mnemonic based keybindings AND physical-key based keybindings... but this would require extra disambiguation code below.

Reply:
What extra code would be needed to store both values? It could be useful to store both, maybe it will help with migrating the key codes to key names in the app too.

Reply:
Thank you for the pushback.